### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by seuros**
Cleanup outdated documentation/config
Enable MFA for publishing the gem.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 name: Tests
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```
Comment (keithdoggett) [inline .github/workflows/tests.yml:9]: We need to run tests on all PRs

**Review by keithdoggett — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (seuros) [reply .github/workflows/tests.yml:9]: Yeah . This make sense.

Comment (keithdoggett) [reply .github/workflows/tests.yml:9]: The block should look like this then

```
on:
  push:
    branches:
      - master
  pull_request:
```

Right now it's only running PRs on branches that are merging with `master` I believe. Tests didn't run on this PR yet.

**Comment by seuros** [comment]
@keithdoggett  this PR fix the tasks issue too.

**Review by keithdoggett — APPROVED** [review]
LGTM thank you for the continued work on this

**Pull request merged**